client/client.go

```diff
@@ -137,8 +136,14 @@ func NewNotaryRepository(baseDir, gun, baseURL string, rt http.RoundTripper,
 
 // Initialize creates a new repository by using rootKey as the root Key for the
 // TUF repository.
-func (r *NotaryRepository) Initialize(uCryptoService *cryptoservice.UnlockedCryptoService) error {
-	rootCert, err := uCryptoService.GenerateCertificate(r.gun)
+func (r *NotaryRepository) Initialize(rootKeyID string) error {
+	privKey, _, err := r.KeyStoreManager.KeyStore.GetKey(rootKeyID)
```

Comment:
In the interests of cryptoservices supporting multiple keystores: `r.cryptoservice.GetKey(rootKeyID)`